examples/dbscan/dbscan.cpp

```diff
@@ -486,11 +486,14 @@ int main(int argc, char *argv[])
   printf("#cluster points : %d [%.2f%%]\n", num_cluster_points,
          (100.f * num_cluster_points / data.size()));
 
+  bool success = true;
   if (verify)
   {
     auto passed = ArborX::Details::verifyDBSCAN(exec_space, primitives, eps,
                                                 core_min_size, labels);
     printf("Verification %s\n", (passed ? "passed" : "failed"));
+    if (!passed)
+      success = false;
```

Comment:
```suggestion
    success = ArborX::Details::verifyDBSCAN(exec_space, primitives, eps,
                                            core_min_size, labels);
    printf("Verification %s\n", (success ? "passed" : "failed"));
```

Reply:
don't agree with the suggestion

Reply:
Why not?

Reply:
Because I don't find your suggestion more readable than what I proposed and that I don't feel like re-running the CI.
That said you own that file so if you want to insist you can go ahead and commit the suggestion yourself.